test/Microsoft.AspNetCore.WebUtilities.Tests/Base64UrlTextEncoderTests.cs

```diff
@@ -65,5 +70,26 @@ public void DecodeToBase64String_ReturnsValid_Base64String(string text, string e
             // Assert
             Assert.Equal(expectedValue, actual);
         }
+
+        [Fact]
+        public void UrlEncode_Base64EncodedData_DoesNotChangeData()
+        {
+            for (int length = 0; length != 256; ++length)
+            {
+                // Arrange
+                var data = new byte[length];
+                for (int index = 0; index != length; ++index)
+                {
+                    data[index] = (byte)(5 + length + (index * 23));
+                }
+                string text = Base64UrlTextEncoder.Encode(data);
+
+                // Act
+                var urlEncodedText = UrlEncoder.Default.Encode(text);
+
+                // Assert
+                Assert.Equal(text, urlEncodedText);
```

Comment:
Do we care here? UrlEncoder might choose to do no-op since it couldn't find any characters to encode. We only want to make sure that characters remain the same.